chart/environment-values/values.custom.tpl.yaml

```diff
@@ -38,43 +40,48 @@ owfms:
     owfms:
       annotations:
         external-dns.alpha.kubernetes.io/hostname: fms-${NAMESPACE}.${DOMAIN}
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${CERTIFICATE_ARN}
   public_env_variables:
     OWSEC: sec-${NAMESPACE}.${DOMAIN}:16001
-owgwui:
-  ingresses:
-    default:
-      hosts:
-      - webui-${NAMESPACE}.${DOMAIN}
-      annotations:
-        external-dns.alpha.kubernetes.io/hostname: webui-${NAMESPACE}.${DOMAIN}
-  public_env_variables:
-    REACT_APP_UCENTRALSEC_URL: https://sec-${NAMESPACE}.${DOMAIN}:16001
 owprov:
   services:
     owprov:
       annotations:
         external-dns.alpha.kubernetes.io/hostname: prov-${NAMESPACE}.${DOMAIN}
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${CERTIFICATE_ARN}
   configProperties:
     openwifi.system.uri.public: https://prov-${NAMESPACE}.${DOMAIN}:16005
     openwifi.system.uri.private: $INTERNAL_RESTAPI_ENDPOINT_SCHEMA://owprov-owprov:17005
     openwifi.system.uri.ui: https://provui-${NAMESPACE}.${DOMAIN}
     iptocountry.ipinfo.token: "${IPTOCOUNTRY_IPINFO_TOKEN}"
   public_env_variables:
     OWSEC: sec-${NAMESPACE}.${DOMAIN}:16001
+owgwui:
```

Comment:
Why `owgwui` added here?

Reply:
I see it is being moved.

Reply:
just moved it from a little up in this file.